tests/debugging/exploration/test_bootstrap.py

```diff
@@ -45,6 +45,7 @@
 """
 
 
+@pytest.mark.skip(reason="Flaky test")
```

Comment:
Are these tests flaky or just broken for this branch?

Reply:
They're broken for this branch, reviewing that with calm I'm seeing that it was fixed on this backport https://github.com/DataDog/dd-trace-py/pull/6353 but it was closed, Do you know why? @mabdinur @majorgreys

Reply:
I believe because we were no longer maintaining 1.14. Is there a reason why such an old release branch is getting updated?